vue-frontend/src/components/home-new/ClientReview.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <div class="tw-container tw-mt-[8rem] tw-ml-auto tw-mx-auto">
+    <div class="swiper-content">
+      <swiper
+        :slidesPerView="1"
+        :centeredSlides="true"
+        :autoplay="{
+          delay: 3000,
+          disableOnInteraction: false,
+        }"
+        :loop="true"
+        :loopedSlides="50"
+        :spaceBetween="20"
+        :pagination="pagination"
+        class="swiper-container"
+      >
+        <swiper-slide v-for="(client, index) in clients" :key="index">
+          <div
+            class="tw-mx-auto sm:tw-px-10 md:tw-px-6 lg:tw-px-2 xl:tw-px-24 2xl:tw-px-40"
+            :class="[client.className]"
+          >
+            <div
+              class="tw-font-roboto-medium tw-text-[1.375rem] md:tw-text-[2.25rem] lg:tw-text-[3.125rem] tw-leading-[1.875rem] md:tw-leading-[2.75rem] lg:tw-leading-[3.75rem]"
+              v-html="client.review"
+            ></div>
+            <div
+              class="tw-flex tw-flex-row tw-justify-between tw-mt-11 md:tw-mt-20"
+            >
+              <div class="tw-flex tw-flex-col">
+                <span
+                  class="v2-canopas-gradient-text tw-font-inter-semibold tw-text-[1rem] md:tw-text-[1.625rem] tw-leading-[1.375rem]"
+                  >{{ client.name }}</span
+                >
+                <span
+                  class="tw-font-inter-light tw-text-[0.75rem] md:tw-text-[1.25rem] tw-leading-[1.375rem] tw-text-black-core"
```

Comment:
font-color is not pure black anywhere. Also match font-color of all divs. Its not same as figma